photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/search/GaussianProcessSearch.scala

```diff
@@ -170,21 +170,25 @@ class GaussianProcessSearch[T](
   }
 
   /**
-   * Selects the best candidate according to the predicted values, where "best" is determined by the given evaluator
+   * Selects the best candidate according to the predicted values, where "best" is determined by the given transformation
    *
    * @param candidates matrix of candidates
    * @param predictions predicted values for each candidate
+   * @param transformation prediction transformation function
```

Comment:
Might be better to add a more detailed comment to explain this. As in, in the case of EI, we always search for the max, but for CB, we search for max / min given the optimization problem.